src/app/timetable/components/Timetable.module.scss

```
@@ -104,6 +104,62 @@
   }
 }
 
+.popoverLayout {
+  background: #ffffff;
+  padding: 20px;
+  z-index: 100;
+  width: 300px;
+  max-height: 500px;
+  border-radius: 10px;
+  border: none;
+  box-shadow: 0px 8px 12px rgba(0, 0, 0, 0.3);
+  color: #333;
+  overflow-y: auto;
+  box-sizing: border-box;
+}
+
+.buttonLayout {
+  display: flex;
+  flex-direction: row-reverse;
+  margin-bottom: 4px;
+}
+
+.closeButton {
+  width: 16px;
+  height: 16px;
+  background-color: inherit;
+  border: 0;
+  cursor: pointer;
+}
+
+.scheduleLayout {
+  display: flex;
+  height: 20px;
+  align-items: center;
+}
+
+.scheduleIcon {
+  height: 20px;
+  width: 20px;
+  max-width: 100%;
+}
+
+.scheduleContent {
+  line-height: 100%;
+  padding-left: 4px;
+}
+
+.popoverTitle {
+  font-size: 24px;
+  margin-bottom: 8px;
+  word-break: break-all;
+}
+
+.popoverDescription {
+  margin-top: 10px;
+  font-size: 14px;
```

```suggestion
  margin-top: 10px;
  font-size: 14px;
   word-break: break-all;
  overflow: auto;
  max-height: 150px;
```
이런 방식을 녹여본다면 content 요소 안에서 세로로 스크롤이 생깁니다!

---

content 세로 스크롤, 제목이 너무 길면 ` text-overflow: ellipsis;`추가 했습니다.